spec/models/revenue_estimation_spec.rb

```diff
@@ -47,5 +48,17 @@
         expect(subject.date.month).to eq(10)
       end
     end
+     
+    context "invalid" do
```

Comment:
This is also a redundant expectation since you already wrote the shoulda matcher test for it. Check line 22. For your growth.